providers/manage.rb

```diff
@@ -178,3 +180,18 @@ def manage_home_files?(home_dir, user)
     true
   end
 end
+
+# This is a workaround to include recipe 'chef-vault' in LWRP.
+def chef_vault_item(bag, item)
```

Comment:
It also appears to work if you just `include ChefVaultItem` rather than copying the contents of the chef_vault_item function here.